app/packages/looker-3d/src/renderables/pcd/shaders.tsx

```diff
@@ -11,34 +11,33 @@ export type ShaderProps = {
   isPointSizeAttenuated: boolean;
   opacity?: number;
   upVector?: THREE.Vector3;
+  pcdType?: "intensity" | "rgb";
```

Comment:
nit:

is this the shader type? A PCD can have both rgb and intensity.

Reply:
> A PCD can have both rgb and intensity.

not true, turns out

Reply:
when it's intensity schema is 
x y z intensity
f f f f

when it's our old way of doing intensity (now we know it's wrong):
x y z rgb
f f f f

and we use rgb.r